### PR TITLE
doc: tweak 2.2 release notes

### PR DESCRIPTION
**Pull request by dbkinder**
Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/release_notes/release_notes_2.2.rst [hunk 1]
```diff
@@ -40,25 +40,26 @@ Elkhart Lake and Tiger Lake processor support.
   <https://newsroom.intel.com/press-kits/intel-industrial-summit-2020>`_,
   Intel announced the latest additions to their
   enhanced-for-IoT Edge portfolio: the Intel® Atom® x6000E Series, Intel®
-  Pentium® and Intel® Celeron® N and J Series (all code named Elkhart Lake),
-  and 11th Gen Intel® Core™ processors (code named Tiger Lake-UP3). The ACRN
+  Pentium® and Intel® Celeron® N and J Series (all codenamed Elkhart Lake),
```
Comment (gvancuts) [inline doc/release_notes/release_notes_2.2.rst:43]: Shouldn't we also be using the replacement for the reg, TM. etc. signed?

**Review by gvancuts — APPROVED** [review]

File: doc/release_notes/release_notes_2.2.rst (hunk 1 above)
Comment (dbkinder) [reply doc/release_notes/release_notes_2.2.rst:43]: Using either ® or |reg| works OK.  But you're right, I lean towards removing non-ASCII characters from the reST sources unless necessary.  I'll change them here.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**